app/templates/_package.json

```diff
@@ -14,7 +14,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo && <% if (nyc) { %>nyc <% if (coveralls) { %>--reporter=lcov --reporter=text <% } %><% } %>ava"
```

Comment:
> I think a "coverage" script could be confusing to some new contributors that might think they need to use it to get coverage somehow.

Could always prefix it: `"_coveralls"`.

-

>  but it also keeps Travis complexities encapsulated in the Travis file.

Not really, now my test script is ugly with flags. Sorry, my opinion is purely based on aesthetics. I love the simplicity of `xo && nyc ava` 😝

Does `nyc` support using the package.json config for reporters?

> We have seen users try to cover two individual actions with two nyc commands and then generate a report - and get confused because the second nyc command deletes the results of the first.

I hate having to do stuff because of user stupidity... I kinda think `nyc` should somehow warn/error on such case.

-

All that being said, I'm ok with using flags if there's no better solution.